src/OpenTelemetry.ResourceDetectors.AWS/README.md

```diff
@@ -1,5 +1,8 @@
 # AWS Resource Detectors
 
+[![nuget](https://img.shields.io/nuget/v/OpenTelemetry.ResourceDetectors.AWS.svg)](https://www.nuget.org/packages/OpenTelemetry.ResourceDetectors.AWS)
```

Comment:
It couldn't find the package, is it expected?

Reply:
This package was not yet released. It is expected.